drake/systems/plants/KinematicsCache.h

```diff
@@ -121,6 +126,44 @@ class KinematicsCache
     }
   }
 
+  template <typename Derived>
+  Eigen::Matrix<typename Derived::Scalar, Derived::RowsAtCompileTime, Eigen::Dynamic> transformVelocityMappingToPositionDotMapping(const Eigen::MatrixBase<Derived>& mat) const
```

Comment:
also, why do you have this taking in row vectors instead of column vectors?  (to call this for a single v to qdot, I have to transpose before and after the call)